src/go/rpk/pkg/cli/cmd/root.go

```diff
@@ -425,3 +430,45 @@ func walk(c *cobra.Command, f func(*cobra.Command)) {
 		walk(c, f)
 	}
 }
+
+func wrappedLocalFlagUsages(cmd *cobra.Command) string {
+	width := 80
+	if ws, err := mTerm.GetWinsize(0); err == nil {
+		width = int(ws.Width)
+	}
+	return cmd.LocalFlags().FlagUsagesWrapped(width - 1)
+}
+
+func wrappedGlobalFlagUsages(cmd *cobra.Command) string {
+	width := 80
+	if ws, err := mTerm.GetWinsize(0); err == nil {
+		width = int(ws.Width)
```

Comment:
Here and above, if width < 80, should we bail / return 80? What does this look like if I make my terminal extremely narrow?

Reply:
With width == 45:
![image](https://user-images.githubusercontent.com/59714880/203333035-21d6b67c-85a0-478e-b9fc-3a40162e4978.png)

width == 45 but wrapping @ 80:
![image](https://user-images.githubusercontent.com/59714880/203333505-db6deb67-3b0a-4121-80ed-d5fd6a48641e.png)

without the change:
![image](https://user-images.githubusercontent.com/59714880/203333138-2619a261-db76-4694-a15a-5d6f8d019bd2.png)


I think we could benefit from it for users with narrow terminals.